modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -174,6 +174,14 @@ image::dashboard-built-in-infrastructure.png[Example: Infrastructure Dashboard]
 The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
 |===
 
+=== Charts with Endpoints
+
+Charts that contain endpoint metrics sort the available endpoints in a menu.
+The requested time period for the results shown in the endpoint charts may include multiple grouping intervals because the data is ingested in intervals of time.
```

Comment:
"can include" instead of "may include"
Is there another word you can use besides "ingested"? Maybe "processed"?

Reply:
```suggestion
The requested time period for the results shown in the endpoint charts can include multiple grouping intervals because the data is processed in intervals of time.
```